### PR TITLE
Unit tests infrastructure

### LINKED ISSUE #6: Infrastructure for unit tests
- Ability to have `*.spec.js` with BEM imports.
- An example of some unit-tests runner setup.

### PR DESCRIPTION
**Pull request by awinogradov**
Close #6

### CODE REVIEW

**Comment by awinogradov** [comment]
@veged can you activate Travis here https://travis-ci.org/bem/bem-react-core?

**Comment by dfilatov** [comment]
@awinogradov did it )

**Comment by awinogradov** [comment]
👍

**Comment by awinogradov** [comment]
@dfilatov it works ;)

**Comment by awinogradov** [comment]
Ready to merge) But we need issue with list of required test-suits for bem-react-core.

File: __tests__/simple.spec.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { shallow } from 'enzyme';
+import MyBlock from 'b:MyBlock';
+
+
+it('MyBlock should be a', () => {
+  const wrapper = shallow(
+    <MyBlock />
+  );
+
+  expect(wrapper.type()).toBe('a');
```
Comment (veged) [inline __tests__/simple.spec.js]: is `expect` a magic variable?

**Comment by veged** [comment]
what about https://github.com/bem/bem-react-core/tree/master/tests?

File: __tests__/simple.spec.js (hunk 1 above)
Comment (awinogradov) [inline __tests__/simple.spec.js]: Yes, it's global from Jest.

**Pull request merged**